tests/algorithms/test_bounded_mean/test_bounded_mean_int64_data.proto

```diff
@@ -1,2 +1,2 @@
-
-;type.googleapis.com/differential_privacy.BoundedMeanSummary
+M
+;type.googleapis.com/differential_privacy.BoundedMeanSummary�ʵ���	
```

Comment:
some extra characters are added to the file. Verify if this is really needed?

Reply:
This is needed for `test_bounded_mean_int64`.

This file has a very confusing name. It's called `test_bounded_mean_int64_data.proto`, but it doesn't store a protobuf message type definition, instead it actually stores some serialized data. The reason Madhava's PR was not passing checks was because this thing was missing here. If you restore the old version of this file (remove weird characters), you will see `test_bounded_mean_int64` failing again.